code/game/objects/structures/arclumin.dm

```diff
@@ -0,0 +1,21 @@
+/*
+
+At 7:30 AM on March 8th, 2017, ArcLumin died in a fatal car crash on impact.
```

Comment:
@tgstation/commit-access 

What is the final decision on this? Feels weird having reference to someone who was never part of our community and I feel like an asshole for saying it